### ISSUE TITLE
enchance number filter input

### ISSUE THREAD

**Issue by bo-io-gov-mo** [bo-io-gov-mo/website#75]
now only can input numeric:  1,2,3... 
can enhance to: 1/2016, 3/1999, 122/84/M...?

http://betan.io.gov.mo/cn/legis/list/a/46
![image](https://cloud.githubusercontent.com/assets/19220046/18297006/2de312fe-74e0-11e6-8829-53a018c957c2.png)

**Comment by xiquinho** [comment]
this  was supposed to be a simple list. Now you want enhance, tomorrow you will want autocomplete after tomorrow want sort and in the end I have to build and maintain two search pages just like I'm saying from the beginning.

**Comment by bo-io-gov-mo** [comment]
yes. it is basic/simple function of legis search. It is separate from full text search,  from your saying is right  it is two search.

**Comment by bo-io-gov-mo** [comment]
not urgent.
